graphql_api/mutation/labeling_task_label.py

```diff
@@ -83,6 +84,35 @@ def mutate(
         return UpdateLabelingTaskLabelColor(ok=True)
 
 
+class UpdateLabelingTaskLabelName(graphene.Mutation):
+    class Arguments:
+        project_id = graphene.ID(required=True)
+        labeling_task_label_id = graphene.ID(required=True)
+        new_name = graphene.String(required=True)
+
+    ok = graphene.Boolean()
+
+    def mutate(self, info, project_id: str, labeling_task_label_id: str, new_name: str):
+        auth.check_demo_access(info)
+        auth.check_project_access(info, project_id)
+        manager.update_label_name(project_id, labeling_task_label_id, new_name)
+        return UpdateLabelingTaskLabelColor(ok=True)
+
+
+class HandleLabelReNameWarnings(graphene.Mutation):
```

Comment:
Should this be 'HandleLabelRenameWarnings'? For me, ReName is confusing while reading